nginx.montagu.conf

```diff
@@ -38,8 +38,16 @@ server {
     location /resources/ {
     }
 
-    # Eventually the visualisation app, not authenticated
-    location /visualisation {
+    # Private data visualisation app - authenticated with Montagu via OrderlyWeb, redirects to the report on OW
+    rewrite /2020/visualization-pro /2020/visualisation-pro/ last;
+    location /2020/visualisation-pro/ {
+        proxy_pass  http://orderly_web_web:8888/report/internal-2018-interactive-plotting/version/20191114-152933-6918620a/artefacts/index.html?inline=true;
```

Comment:
You can test this locally by changing this line to something that will exist (ie not any particularly report version) e.g. 
`proxy_pass  http://orderly_web_web:8888/?;`